projects/ad7606x_fmc/Readme.md

```diff
@@ -1,46 +1,36 @@
+  * [EVAL-AD7609](https://www.analog.com/en/resources/evaluation-hardware-and-software/evaluation-boards-kits/eval-ad7609.html)
+- System documentation: https://wiki.analog.com/resources/eval/user-guides/ad7606x-fmcz
+- HDL project documentation:https://analogdevicesinc.github.io/hdl/projects/ad7606x_fmc/index.html
 
-IMPORTANT: Set AD7606X device model, ADC Read Mode option and external clock option
+## Supported parts
 
-How to use over-writable parameters from the environment:
-```
-hdl/projects/ad7606x_fmc/zed> make INTF=1 NUM_OF_SDI=4
-INTF - Defines the operation interface: 0 - Parallel, 1 - Serial
-NUM_OF_SDI - Defines the number of SDI lines used: 1, 2, 4, 8
-ADC_N_BITS - Specifies the ADC resolution: 16 or 18 bits; only for the Parallel Interface
+| Part name                                                    | Description                                                           |
+|--------------------------------------------------------------|-----------------------------------------------------------------------|
+| [AD7606B](https://www.analog.com/en/products/ad7606b)        | 8 Channels, 16-bit, 800 kSPS Bipolar Input, Simultaneous sampling ADC |
+| [AD7606C-16](https://www.analog.com/en/products/ad7606c-16)  | 8 Channels, 16-bit, 1 MSPS Bipolar Input, Simultaneous sampling ADC   |
+| [AD7606C-18](https://www.analog.com/en/products/ad7606c-18.) | 8 Channels, 18-bit, 1 MSPS Bipolar Input, Simultaneous sampling ADC   |
+| [AD7605-4](https://www.analog.com/en/products/ad7605-4)      | 4 Channels, 16-bit, 300 kSPS Bipolar Input, Simultaneous sampling ADC |
+| [AD7606](https://www.analog.com/en/products/ad7606)          | 8 Channels, 16-bit, 200 kSPS Bipolar Input, Simultaneous sampling ADC |
+| [AD7606-6](https://www.analog.com/en/products/ad7606-6)      | 6 Channels, 16-bit, 200 kSPS Bipolar Input, Simultaneous sampling ADC |
+| [AD7606-4](https://www.analog.com/en/products/ad7606-4)      | 4 Channels, 16-bit, 200 kSPS Bipolar Input, Simultaneous sampling ADC |
+| [AD7607](https://www.analog.com/en/products/ad7607)          | 8 Channels, 14-bit, 200 kSPS Bipolar Input, Simultaneous sampling ADC |
+| [AD7608](https://www.analog.com/en/products/ad7608)          | 8 Channels, 18-bit, 200 kSPS Bipolar Input, Simultaneous sampling ADC |
+| [AD7609](https://www.analog.com/en/products/ad7609)          | 8 Channels, 18-bit, 200 kSPS Bipolar Input, Simultaneous sampling ADC |
 
-For the serial interface, the following parameters will be used in make command: INTF, NUM_OF_SDI.
-For the parallel interface, the following parameters will be used in make command: INTF, ADC_N_BITS.
+  * Project Doc: 
+  * HDL Doc: https://wiki.analog.com/resources/eval/user-guides/ad7606x-fmc/hdl
+  * NO-OS Drivers: [AD7606 - No-OS Driver](https://wiki.analog.com/resources/tools-software/uc-drivers/ad7606)
+  * Linux Drivers: https://wiki.analog.com/resources/tools-software/linux-drivers/iio-adc/axi-adc-hdl
```

Comment:
```suggestion
```

These shouldn't be in the main README. They belong to the carrier specific README.